src/camps/models.py

```diff
@@ -5,9 +5,8 @@
 from django.core.exceptions import ValidationError
 from django.db import models
 from django.urls import reverse
-from psycopg2.extras import DateTimeTZRange
-
 from program.models import EventLocation, EventType
+from psycopg2.extras import DateTimeTZRange
```

Comment:
I once made the Bornhack website run in SQLite3 (for development purposes) but was lead to understand that it is 100% tied to Postgresql, I guess there's no way out? :)

Reply:
no we made the choice to go PG only a while back :)